test/driver.cpp

```diff
@@ -23,66 +23,82 @@
 
 #include "tester.h"
 
+using std::string;
+using std::string_view;
+using std::size_t;
+
 //must be defined in a unit-specific source file such as "array-test.cpp"
 void runTests();
 
-void processCmdLine(std::vector<std::string_view> arguments);
+void processCmdLine(char* arguments[], const size_t size);
+
+void replace_all(string& str, const string& substr, 
+   const string& toReplace);
```

Comment:
I suggest changing the parameter names in this function declaration to match the names used in the function's definition.

```suggestion
void replace_all(string& str, const string& substr, 
   const string& new_substr);
```

Reply:
Agreed! Oversight on my part.